src/main/target/CC3D/target.c

```diff
@@ -26,17 +26,19 @@
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
 
-    DEF_TIM(TIM4, CH1, PB6, TIM_USE_PWM,               0),   // S1_IN/
+//    DEF_TIM(TIM4, CH1, PB6, TIM_USE_PWM,               0),   // S1_IN/  <--Moved to allow Parallel PWM with proper ESC
     DEF_TIM(TIM3, CH2, PB5, TIM_USE_PWM,               0),   // S2_IN - SoftSerial TX - GPIO_PartialRemap_TIM3 / Sonar trigger
     DEF_TIM(TIM3, CH3, PB0, TIM_USE_PWM,               0),   // S3_IN - SoftSerial RX / Sonar echo / RSSI ADC
     DEF_TIM(TIM3, CH4, PB1, TIM_USE_PWM,               0),   // S4_IN - Current
     DEF_TIM(TIM2, CH1, PA0, TIM_USE_PWM,               0),   // S5_IN - Vbattery
-    DEF_TIM(TIM2, CH2, PA1, TIM_USE_PWM | TIM_USE_PPM, 0),   // S6_IN - PPM IN
+    DEF_TIM(TIM2, CH2, PA1, TIM_USE_PWM| TIM_USE_PPM,  0),   // S6_IN - PPM IN
+    DEF_TIM(TIM3, CH1, PB4, TIM_USE_PWM,               1), // S5_OUT - GPIO_PartialRemap_TIM3 - LED Strip
```

Comment:
Looking at this now, it is probably better to move this to the front of the PWM list (where you commented out PB6). Motor / PWM channel numbers are assigned in order of appearance in this list, and if you replace the PB6 entry with PB4 in the same place, PWM users will only have to move one wire, instead of shifting all of them by one.

Reply:
It is actually quite the contrary.  If you leave it in, then user will have to shift their pins if you look at the pin definitions (found on Cleanflight wiki):
![cc3d pin def](https://user-images.githubusercontent.com/936003/30589021-f3856346-9d06-11e7-8f51-88eee7c96d41.JPG)

I know, because when I tested I was expecting to have to shift but I did not have to do any thing.  So now that Pin 3 on the connector becomes a motor pin instead of an RX pin.

Reply:
Ok, We'll leave it as it is then, to match the spec. Users with existing wiring can always use resource remapping to change it.

Reply:
Also, if you bump the Timer up in the list it will then mandate that a wire needs to be used to connect an ESC out pin to the reciever.  Leaving it the way I have it gives the option to only have to use the wires out of the PWM In connector (or connector 1).